backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -80,7 +82,7 @@
 public class EndpointResource {
 
     private static final List<EndpointType> systemEndpointType = List.of(
-            EndpointType.EMAIL_SUBSCRIPTION
+            EndpointType.EMAIL_SUBSCRIPTION, EndpointType.DRAWER
     );
```

Comment:
This list isn't needed anymore since `EndpointType` now contains the information about system endpoint types.